styx-scheduler-service/src/main/java/com/spotify/styx/ServiceAccountKeyManager.java

```diff
@@ -88,7 +88,7 @@ public void tryDeleteKey(String keyName) {
     try {
       deleteKey(keyName);
     } catch (IOException e) {
-      LOG.debug("Ignoring error while deleting key {}", keyName, e);
+      LOG.debug("Ignoring error while deleting key {} {}", keyName, e);
```

Comment:
This shouldn't be needed I think. You might did filtering on stackdriver log I guess, but that wouldn't give you the following line (which is the stacktrace in this case).

Reply:
Ah you're totally right